.prettierrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+    semi: false,
+    singleQuote: false,
+    printWidth: 120,
+    tabWidth: 4,
```

Comment:
I strongly believe tabs are better than spaces - it's not a code style choice for me, but an accessibility choice.
Some programmers with impaired vision need the option to scale the tab-wdth up or down so they can better navigate the code at a glance. Tabs gives us this.
However, I am not sure if setting a fixed width of 4 spaces for tabs will prevent this. Do you know? What are your thoughts on this?

Reply:
There are definitely two schools of thought on this matter. Personally, I go with either way as I usually include EditorConfig in my projects which will automatically set my editor settings to comply with project.

Hard tabs or spaces with a width of 4 is usually a good choice unless the logic includes a lot of nesting (which is usually a code smell, of course).